packages/yew/src/functional/hooks/use_memo.rs

```diff
@@ -0,0 +1,39 @@
+use crate::functional::{hook, use_hook};
+use std::rc::Rc;
+
+/// This hook is used for obtaining a immutable reference to a value that is recalculated when it's
+/// dependency changes.
```

Comment:
just a suggestion, but might be worse in the end :D
```suggestion
/// Get a immutable reference to a memoized value
/// Memoization means it will only get recalculated when provided dependencies update/change
```